.cloudbees/workflows/test.yaml

```diff
@@ -0,0 +1,45 @@
+name: ecr-promote-image
+kind: workflow
+apiVersion: automation.cloudbees.io/v1alpha1
+
+on:
+  push:
+    branches:
+      - '**'
+
+jobs:
+  ec2-promote-image-job:
+    steps:
+      - name: Configure aws credentials
+        uses: cloudbees-io/configure-aws-credentials@v1
+        with:
+          aws-region: us-east-1
+          aws-access-key-id: ${{ secrets.CB_ACTIONS_TEST_AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.CB_ACTIONS_TEST_AWS_SECRET_KEY }}
+
+      - id: ecr-promote-image
+        name: ecr promote image
+        uses: cloudbees-io/ecr-promote-image@v1
+        with:
+          registry-url: 020229604682.dkr.ecr.us-east-1.amazonaws.com
+          source-repository-name: actions/trufflehog-secret-actions
+          target-repository-name: throwaway/actions/trufflehog-secret-actions
+          source-tag: 'CBP-1914-401e607a2c93dc136a6db5e84ae31ce46a276c5c'
+          target-tag: 'promote-test'
+
+      - name: Notify failures
+        if: ${{ failure() }}
+        uses: ./actions/notify-slack-action
+        with:
+          webhook-url: ${{ secrets.CBP_BT_SLACK_WEBHOOK_URL }}
+          # action name (as per cloudbees-io) and this file name must be action name with .yaml extension
+          # this agreement is a prerequisite to send workflow urls in slack notifications in case of failure
+          action-name: ecr-promote-image
+
+      - name: Verify artifact creation
```

Comment:
Lets move this step `Verify artifact creation` to be before ` Notify failures`. That way if there is any issue in accessing the output parameter, that will also be notified on.